app/redux/reducers/AppStateReducer.js

```diff
@@ -20,6 +23,14 @@ const AppStateReducer = (state = defaultState, action) => {
             return Object.assign({}, state, {
                 tabIndex: action.tabIndex,
             });
+        case DISABLE_TAB_SWIPE:
+            return Object.assign({}, state, {
+                tabSwipeEnabled: false,
+            });
+        case CHANGE_SLIDER_VELOCITY:
```

Comment:
- [ ] Assuming you believe this to be required, why are you using CHANGE_SLIDER_VELOCITY instead of ENABLE_TAB_SWIPE